src/components/WebEditor/tabs.jsx

```diff
@@ -44,63 +75,98 @@ function a11yProps(index) {
 @params:
-${inputs?.map((inp,idx) => `${inp} = ${JSON.stringify(data?.inputs[idx])}`).join('\n  ')}
+${'  ' + inputs?.map((inp, idx) => `${inp} = ${JSON.stringify(data?.inputs[idx])}`).join('\n  ')}
 */
 const func = (${String(inputs)}) => {
 // Write the code here
 return 'hello world'
 }
-    `)
-  }, [value, problems])
-
+    `
+    );
+  }, [value, problems, setExp, setJs]);
 
   const handleChange = (event, newValue) => {
     setValue(newValue);
   };
 
+  const toggleHint = () => {
+    setShowHint(!showHint);
+  };
+
   return (
     <Box sx={{ width: '100%' }}>
       <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
         <Tabs value={value} onChange={handleChange} aria-label="basic tabs example">
-        {problems.map((element, idx) => {
-            // return <Tab label=`Item ${idx}` {...a11yProps(0)} />
-            let label = `Problem ${idx+1}`
-            return <Tab label={label} {...a11yProps(idx)} />
-        })
-        }
+          {problems.map((element, idx) => {
+            let label = `Problem ${idx + 1}`;
+            let difficulty = problems[idx]?.difficulty || 'default';
+            let tabColor = getLighterColor(getDifficultyColor(difficulty), 0.6); // Adjust brightness with alpha function
+            return (
+              <Tab
+                key={idx}
+                label={label}
+                {...a11yProps(idx)} // Correct usage of a11yProps
+                sx={{ backgroundColor: tabColor }}
+              />
+            );
+          })}
         </Tabs>
-        <div className='problem-statement'>
-            <h4>{problems[value]?.['problem name']}</h4>
-            <span>{problems[value]?.['description']}</span>
-            <h5 style={{fontStyle: 'italic'}}>Input Format</h5>
-            <span>{problems[value]?.['inputs description']}</span>
-            <h5 style={{fontStyle: 'italic'}}>Output Format</h5>
-            <span>{problems[value]?.['expected description']}</span>
+
+        <div className="problem-statement">
+          <h4 style={{ marginBottom: '9px' }}>{problems[value]?.['problem name']}</h4>
+          <DifficultyChip
+            label={problems[value]?.['difficulty']}
+            style={{ backgroundColor: getDifficultyColor(problems[value]?.['difficulty']) }}
+          />
+          <br />
+          <span>{problems[value]?.['description']}</span>
+          <h5 style={{ fontStyle: 'italic' }}>Input Format</h5>
+          <span>{problems[value]?.['inputs description']}</span>
+          <h5 style={{ fontStyle: 'italic' }}>Output Format</h5>
+          <span>{problems[value]?.['expected description']}</span>
+          <br />
+          <Button
+            style={{ margin: '20px 0px' }}
+            onClick={toggleHint}
+            sx={{
+              mt: 1,
+              textTransform: 'none',
+              fontWeight: 'bold',
+              color: '#000', // Black text color
+              backgroundColor: '#f0f0f0', // Light gray background color
```

Comment:
Instead of putting the comments here, you can create mnemonic consts for these color codes. That way they will also become reusable

Reply:
Thank you for the feedback. I have created mnemonic constants for the color codes and removed the build directory. Please review the latest commit to confirm that the changes meet the requirements.